sapi/cli/tests/017.phpt

```diff
@@ -52,17 +61,26 @@ function a_function_with_some_name() {
 a_function_w	);
 EOT;
 
-foreach ($codes as $key => $code) {
-    echo "\n--------------\nSnippet no. $key:\n--------------\n";
-    $proc = proc_open("$php $ini -a", $descriptorspec, $pipes);
-    fwrite($pipes[0], $code);
-    fclose($pipes[0]);
-    proc_close($proc);
-}
-
+runReplCodes($codes);
 echo "\nDone\n";
+
 $dir = PHP_OS_FAMILY == 'Windows' ? getenv("USERPROFILE") : getenv("HOME");
 var_dump(file_exists($dir . '/.php_history'));
+
+define("TMPDIR", __DIR__ . "/");
+$php_history_tmp = TMPDIR . "php_history";
```

Comment:
Please use `$php_history_tmp = tempnam(sys_get_temp_dir(), 'php_history');` to avoid having to clean up the file and putting it in the ignore list.

Reply:
Thanks i've applied the changes, current CI failure doesn't look related